site/docs/Tutorial-Mutations.md

```diff
@@ -87,16 +87,16 @@ var fakeDatabase = {};
 
 // The root provides the top-level API endpoints
 var root = {
-  getMessage: function({author}) {
+  getMessage: function ({author}) {
```

Comment:
That's how I would write it in my own code, but for the tutorial I think it is nice to use minimal fancy ecmascript features so that these docs are approachable to as many people as possible.